spark/src/main/scala/io/delta/tables/execution/DeltaTableOperations.scala

```diff
@@ -112,6 +114,50 @@ trait DeltaTableOperations extends AnalysisHelper { self: DeltaTable =>
     toDataset(sparkSession, restore)
   }
 
+  protected def executeClone(
+      table: DeltaTableV2,
+      target: String,
+      replace: Boolean,
+      properties: Map[String, String],
```

Comment:
I think we should move the `properties` to the end of the list of argument since this is what's happening in `clone`, `cloneAtVersion` and timestamp as well.

Reply:
That would be odd since those are optional, we would have to specify `None` for version and timestamp in all calls to `executeClone`. It already matches the normal pattern of of the `clone*` functions, the timestamp and version are just extra things that are handled differently for each use of `executeClone`.

Reply:
I see, make sense @Kimahriman